### ISSUE TITLE
Makefile is not detecting proper VERSION and failing to download deps (403s)

### ISSUE THREAD

**Issue by sempervictus** [wazuh/wazuh#4773]
|Wazuh version|Component|Install type|Install method|Platform|
|---|---|---|---|---|
| 3.11.4 | Source Build | Agent | Sources | Arch, current |

Attempting to build from source when doing the `make deps` step results in the Makefile downloading HTML 403 responses as tar.gz outputs and then trying to gunzip them (this fails, obviously). The reason for these 403's is that the VERSION declaration in the Makefile produces an incorrect version string:
```
$ make deps 
curl -so external/cJSON.tar.gz https://packages.wazuh.com/deps/3 11./cJSON.tar.gz
```
which should be
```
$ make deps 
curl -so external/cJSON.tar.gz https://packages.wazuh.com/deps/3.11/cJSON.tar.gz
```
The relevant Makefile line is
```
 861 VERSION = $(subst $(space),.,$(wordlist 1,2,$(subst ., ,$(patsubst v%,%,$(shell cat VERSION)))))
```

**Comment by sempervictus** [comment]
Seems to be a make 4.3 thing, but this works just fine:
```
VERSION = $(subst $() $(),.,$(wordlist 1,2,$(subst ., ,$(patsubst v%,%,$(shell cat VERSION)))))
```

**Comment by vikman90** [comment]
Hi @sempervictus,

As I mentioned in the PR #4774, this problem has been merged into 3.14 (8e2f4d8).

Let me close this issue. Thank you very much again for opening this issue and contributing to fixing it.

Best regards.
